roamance/src/components/footer.tsx

```
@@ -105,18 +139,27 @@ export function Footer() {
-            <a href="#" className="text-muted-foreground hover:text-primary transition-colors">
-              <TwitterIcon />
+            <a
+              href="#"
+              className="text-muted-foreground hover:text-primary transition-colors"
+            >
+              <Twitter />
             </a>
-            <a href="#" className="text-muted-foreground hover:text-primary transition-colors">
-              <InstagramIcon />
+            <a
+              href="#"
```

Biome (1.9.4)

[error] Provide a valid value for the attribute href.

The href attribute should be a valid a URL
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)